package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>gz-utils2</name>
+  <version>2.2.0</version>
+  <description>Gazebo Utils : Classes and functions for robot applications</description>
+
+  <maintainer email="addisu@openrobotics.org">Addizu Z. Taddese</maintainer>
+  <license>Apache License 2.0</license>
+
+  <url type="website">https://github.com/gazebosim/gz-utils</url>
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <build_depend>gz-cmake3</build_depend>
```

Comment:
Not a blocker here, but maybe eventually worth adding a key for CLI11 (`libcli11-dev`) and adding a `depend` here, since we are currently vendoring, but could probably drop that.